scripts/vsc-venv-wrapper/vsc-venv.sh

```diff
@@ -0,0 +1,97 @@
+SCRIPT_NAME=${BASH_SOURCE[0]} # $0 cannot be used as it gives '-bash' when sourced
+
+usage() {
+  echo "Usage: source $SCRIPT_NAME {activate <requirements.txt> [modules.sh] | deactivate}"
```

Comment:
i think that this behaviour is not ok.
for a new venv, you need a module.sh to at least set the python version

but i would store the modules.sh and requirements.txt in the venv dir; and the when these are not provided; you reuse that one (well, at least the modules.sh) instead of doing nothing.
having the modules.sh and requiremts.tx you can later also implement a clone command

i also don't understand why the requirements.txt is mandtory: once initialised, to reuse the same environment, you don't need it anymore?

Reply:
I tried to make the script as fool-proof as possible. Copying the `modules.sh` and `requirements.txt` and making it optional could open the user up for unexpected behavior. The user would need to remember to initialize the script with the requirements every time packages are added. 

A solution could be to assume the `requirements.txt` and `modules.sh` are present in the root of the project (which is the case most of the time). That way, the user doesn't need to type out the requirements and modules every time.

Reply:
```suggestion
  echo "Usage: source $SCRIPT_NAME {activate <requirements.txt> [modules.txt] | deactivate}"
```